Sources/ContentScrollView.swift

```diff
@@ -17,6 +17,14 @@ open class ContentScrollView: UIScrollView {
 
     fileprivate var options: SwipeMenuViewOptions.ContentScrollView = SwipeMenuViewOptions.ContentScrollView()
 
+    private var visiblePageViews: [Int: UIView] = [:]
+
+    /// previous page count to setup
+    private let previousBufferPageCount: Int = 1
+
+    /// next page count to setup
+    private let nextBufferPageCount: Int = 1
```

Comment:
I guess there is not much benefit to separate `previous` and `next` preloading page count since it is symmetric in general.

This PR is OK for now, but for future improvement, these values should be unified and be provided as an argument.